watertap/unit_models/tests/test_osmotically_assisted_reverse_osmosis_0D.py

```diff
@@ -1395,7 +1395,7 @@ def test_friction_factor_spiral_wound(self):
             concentration_polarization_type=ConcentrationPolarizationType.calculated,
             mass_transfer_coefficient=MassTransferCoefficient.calculated,
             pressure_change_type=PressureChangeType.calculated,
-            friction_factor=FrictionFactor.spiral_wound,
```

Comment:
I assume with the new config option, friction factor is still being tested since it will change accordingly with module type.

Reply:
Correct. The spiral-wound module type is selected in this test